torchquad/__init__.py

```diff
@@ -7,24 +7,30 @@
 
 # TODO: Currently this is the way to expose to the docs
 # hopefully changes with setup.py
+from .integration.integration_grid import IntegrationGrid
 from .integration.monte_carlo import MonteCarlo
 from .integration.trapezoid import Trapezoid
 from .integration.simpson import Simpson
-from .integration.integration_grid import IntegrationGrid
+from .integration.boole import Boole
+from .integration.vegas import VEGAS
 
 from .plots.plot_convergence import plot_convergence
 from .plots.plot_runtime import plot_runtime
 
 from .utils.enable_cuda import enable_cuda
+from .utils.set_precision import set_precision
 
 __all__ = [
     "IntegrationGrid",
     "MonteCarlo",
     "Trapezoid",
     "Simpson",
+    "Boole",
```

Comment:
Maybe we should report as major that we added boole? I don't see it in the changelog.

Reply:
Done.